src/main/java/main.java

```diff
@@ -0,0 +1,191 @@
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.NoHeadException;
+import org.eclipse.jgit.diff.DiffEntry;
+import org.eclipse.jgit.diff.DiffFormatter;
+import org.eclipse.jgit.lib.ObjectId;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.eclipse.jgit.revwalk.RevWalk;
+import org.eclipse.jgit.treewalk.TreeWalk;
+import org.refactoringminer.api.GitHistoryRefactoringMiner;
+import org.refactoringminer.api.GitService;
+import org.refactoringminer.api.Refactoring;
+import org.refactoringminer.api.RefactoringHandler;
+import org.refactoringminer.rm1.GitHistoryRefactoringMinerImpl;
+import org.refactoringminer.util.GitServiceImpl;
+
+import java.io.*;
+import java.lang.reflect.Array;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import satd_detector.core.train.Train;
+import satd_detector.core.utils.SATDDetector;
+
+
+public class main {
+    static List<String> commitDate = new ArrayList<>();
+    static List<String> commit_include = new ArrayList<>();
+
+
+
+    public static void main(String[] args) {
+        GitService gitService = new GitServiceImpl();
+
+        ProcessBuilder processBuilder = new ProcessBuilder();
+
+        try {
+            Repository repository = gitService.openRepository("aspectj.eclipse.jdt.core");
```

Comment:
対象リポジトリは変わる可能性があるので、かわっても修正が少ないように変数か定数で定義してください。
```suggestion
            Repository repository = gitService.openRepository(TARGET_REPO);
```

Reply:
他の場所も変更する。